custom_components/opnsense/config_flow.py

```
@@ -409,125 +672,127 @@
-                CONF_DEVICE_TRACKER_CONSIDER_HOME, default=device_tracker_consider_home
-            ): vol.All(vol.Coerce(int), vol.Clamp(min=0, max=600)),
-        }
+            self.hass.config_entries.async_update_entry(
+                entry=self.config_entry, data=self._config, options=self._options
+            )
+            return self.async_create_entry(data=self._config)
 
-        return self.async_show_form(step_id="init", data_schema=vol.Schema(base_schema))
+        return self.async_show_form(
+            step_id="init",
+            data_schema=_build_options_init_schema(user_input=user_input, fallback=self._config),
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'fallback' in function call

(E1123)